usaspending_api/submissions/helpers.py

```diff
@@ -6,17 +6,14 @@
 def get_last_closed_submission_date(is_quarter: bool) -> dict:
```

Comment:
I know this isn't part of your ticket, but should we collapse these duplicate functions down before they become too entrenched?  I peeked at how this one is used and it seems like yours would pretty much just plug right in.

Reply:
Done